qemu/tests/cfg/snp_basic_config.cfg

```diff
@@ -0,0 +1,32 @@
+- snp_basic_config:
+    type = snp_basic_config
+    kill_vm = yes
+    login_timeout = 240
+    start_vm = no
+    image_snapshot = yes
+    mem = 8192
+    smp = 8
+    vm_secure_guest_type = snp
+    vm_sev_reduced_phys_bits = 1
+    vm_sev_cbitpos = 51
+    virtio_dev_disable_legacy = on
+    bios_path =  /usr/share/edk2/ovmf/OVMF.amdsev.fd
+    snp_module_path = "/sys/module/kvm_amd/parameters/sev_snp"
+    module_status = Y y 1
+    snp_guest_check = "journalctl|grep -i -w snp"
+    guest_tool_install = "dnf install -y snpguest"
```

Comment:
Has the snpguest package already been supported in RHEL repository?

Reply:
No, snpguest package targets 9.6, so don't need to do a workaround here. To run this case, guest image should install snpguest in prior. Thanks.